### ISSUE TITLE
Document 

### ISSUE THREAD

**Issue by RichardLitt** [orbitdb/orbit-db#405]
[This PR](https://github.com/orbitdb/orbit-db/pull/384#issuecomment-399831839) added `isValidAddress` and `parseAddress` as exposed API functions. We should document these in API.md (as @haadcode noted [here](https://github.com/orbitdb/orbit-db/pull/384#issuecomment-404464184)). 

This should be a fairly quick PR, involving explaining how these work and how to use them.

**Comment by mistakia** [comment]
I’ll gladly handle this. Should I also document the other missing static
methods and properties?

Also there are a few public methods (create & open) that are not aliases
and are not documented.

**Comment by RichardLitt** [comment]
Yeah! That'd be great, @mistakia. More documentation of the other missing static methods and props would be 100.
